docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md

```diff
@@ -13,16 +13,16 @@ An Office 365 developer site makes it easier to get set up and start creating, t
 
 ### Before you start
 
-**Be sure you have an Office 365 subscription that supports a developer site.** If you have one of the following Office 365 subscription plans, you can create a developer site within your existing subscription:
+Unless otherwise noted, developer sites can be created in all plans that include SharePoint, and all SharePoint standalone plans. Including but not limited to:
     
-- Office 365 Midsize Business
-- Office 365 Enterprise E1, E3, E4, E5, or K1
-- Office 365 Education A2, A3, or A4
-- Office 365 Government G1, G3, G4, or K1
+- Microsoft 365 Business Standard
+- Microsoft 365 E3, E5
+- Microsoft 365 A3, A5
+- Microsoft 365 Government G3, G5
 
-**If you have an Office 365 Small Business subscription,** it supports only a single site collection, so you can't create a developer site collection. If you would like to learn more about Office 365 plans for your business, see [SharePoint Online: software boundaries and limits](https://office.microsoft.com/office365-sharepoint-online-enterprise-help/sharepoint-online-software-boundaries-and-limits-HA102694293.aspx).
+If you would like to learn more about Microsoft 365 plans for your business, see [SharePoint Online: software boundaries and limits](https://docs.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits).
     
-For more information about the Office 365 Enterprise offerings, see [Plans &amp; Pricing](https://products.office.com/business/office-365-enterprise-e1-business-software).
+For more information about the Microsoft 365 Enterprise offerings, see [Plans &amp; Pricing](https://www.microsoft.com/en-us/microsoft-365/compare-microsoft-365-enterprise-plans).
```

Comment:
Remove locale from URL and let server redirect based on user's locale.

Reply:
Removed locale